hips/tiles/tile_meta.py

```diff
@@ -0,0 +1,29 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+__all__ = [
+    'HipsTileMeta',
+]
+
+
+class HipsTileMeta:
+    """HiPS tile metadata container.
+
+    This class stores HiPS tile meta attributes.
+
+    Parameters
+    ----------
+    order : `int`
+        Order of HiPS tile
+    ipix : `int`
+        Pixel number of HiPS tile
+    format : `str`
+        Format of the HiPS tile
```

Comment:
Format is ambiguous. Say "file format" in the description.
Also, usually where there's a string that should be one of multiple choices, the common way to document it is this:
```
format : {'fits', 'jpeg'}
    File format
```
i.e. to list the options.